src/main/java/org/apache/maven/plugins/dependency/fromDependencies/CopyDependenciesMojo.java

```diff
@@ -111,6 +109,21 @@ protected void doExecute() throws MojoExecutionException {
         Set<Artifact> artifacts = dss.getResolvedDependencies();
 
         if (!useRepositoryLayout) {
+            Map<String, Integer> copies = new HashMap<>();
+            for (Artifact artifactItem : artifacts) {
+                String destFileName = DependencyUtil.getFormattedFileName(
+                        artifactItem, stripVersion, prependGroupId, useBaseVersion, stripClassifier);
+                int numCopies = copies.getOrDefault(destFileName, 0);
+                copies.put(destFileName, numCopies + 1);
+            }
+            for (Map.Entry<String, Integer> entry : copies.entrySet()) {
+                if (entry.getValue() > 1) {
+                    getLog().warn("Multiple files with the name " + entry.getKey() + " in the dependency tree.");
+                    getLog().warn(
```

Comment:
Is this new line on purpose?

Reply:
It's weird but that's what spotless produces